x-pack/plugins/enterprise_search/common/types/engines.ts

```diff
@@ -20,11 +20,14 @@ export interface EnterpriseSearchEngine {
 }
 
 export interface EnterpriseSearchEngineDetails {
-  indices: EnterpriseSearchEngineIndex[];
+  indices: string[];
   updated_at_millis: number;
   name: string;
 }
-
+export interface EnterpriseSearchEngineDetailsResponse
```

Comment:
introducing this type doesn't make sense to me. I think we should updated `EnterpriseSearchEngineDetails` with `indices: EnterpriseSearchEngineIndex[];` instead of introducing another interface here.